src/track/keyutils.cpp

```diff
@@ -27,8 +27,8 @@ const QRegularExpression s_openKeyRegex(QStringLiteral(
 // Lancelot notation, the numbers 1-12 followed by A (minor) or B(I) (major).
 // or "I", "L", "M", "D", "P", "C" for the advanced modes
 const QRegularExpression s_lancelotKeyRegex(
-        QStringLiteral("\\A(?:^\\s*0*(1[0-2]|[1-9])([ABILMDPC])\\s*$)\\z"));
-constexpr std::string_view s_lancelotMajorModes = "BILM";
+        QStringLiteral("\\A(?:^\\s*0*(1[0-2]|[1-9])([ABILMDPCabilmdpc])\\s*$)\\z"));
+constexpr std::string_view s_lancelotMajorModes = "BILMbilm";
```

Comment:
It might be easier to read to us `toLower()` on line 462. `toLower` will effectively make a single subtraction, while doubling the size might significantly decrease the `find` performance. Likely invisible on our modern CPU, but we need to remain mindful of the slimer specs such as old RPi :smile: